nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/nf-error-handler.js

```diff
@@ -73,6 +73,8 @@
                     $('#message-title').text('Insufficient Permissions');
                 } else if (xhr.status === 409) {
                     $('#message-title').text('Invalid State');
+                } else if (xhr.status === 413) {
```

Comment:
Can response 417 actually be returned by `LimitedContentLengthRequest` in the NiFi server if the content-length header value is different than the actual content length?

Reply:
The associated unit tests exercise various cases where the claim does not match the value.  When the claim is smaller than the transmitted body, the filter allows the request.  When larger, the server will throw an exception and return a 500.